Quadriga/src/main/java/edu/asu/spring/quadriga/dao/impl/projectblog/ProjectBlogEntryDAO.java

```diff
@@ -0,0 +1,74 @@
+package edu.asu.spring.quadriga.dao.impl.projectblog;
+
+import java.util.List;
+import java.util.Properties;
+
+import javax.annotation.Resource;
+
+import org.hibernate.Query;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import edu.asu.spring.quadriga.dao.impl.BaseDAO;
+import edu.asu.spring.quadriga.dao.projectblog.IProjectBlogEntryDAO;
+import edu.asu.spring.quadriga.dto.ProjectBlogEntryDTO;
+import edu.asu.spring.quadriga.exceptions.QuadrigaStorageException;
+
+/**
+ * This class acts as data access object for {@linkplain ProjectBlogEntryDTO} class.
+ * 
+ * @author PawanMahalle
+ *
+ */
+@Service
+public class ProjectBlogEntryDAO extends BaseDAO<ProjectBlogEntryDTO>implements IProjectBlogEntryDAO {
+
+    @Resource(name = "projectconstants")
+    private Properties messages;
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public String getIdPrefix() {
+        return messages.getProperty("projectblogentry_id.prefix");
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public ProjectBlogEntryDTO getProjectBlogEntryDTO(String id) {
```

Comment:
The getProjectBlogEntryDTO() acts as wrapper to generic method getDTO(). The getDTO() needs type of class as method parameter while getProjectBlogEntryDTO() asks for just the id as parameter.